frontend/src/modules/EditZing/Components/EditZing.tsx

```diff
@@ -38,6 +38,16 @@ export const EditZing = () => {
         setShowError(true)
       })
   }, [courseId])
+  const [selectedGroups, setSelectedGroups] = useState<Group[]>([])
+
+  const editSelectedGroups = (group: Group, selected: boolean) => {
+    if (selected) {
+      setSelectedGroups([...selectedGroups, group])
+    } else {
+      setSelectedGroups(selectedGroups.filter((g) => g !== group))
```

Comment:
Having this as a check makes this sort of iffy. See my review below for more details.

Reply:
For the "include" part, [`.some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) might be useful